.github/workflows/windows.yml

```
@@ -11,12 +11,22 @@ jobs:
         ruby: [ mingw, mswin, 3.0, 2.7, 2.6, 2.5, 2.4 ]
     steps:
       - uses: actions/checkout@v2
+      - name: Install libraries with vcpkg
+        run: |
+          vcpkg --triplet x64-windows install libyaml
+      - name: link libraries
+        run: |
+          for %%I in (C:\vcpkg\installed\x64-windows\bin\*.dll) do (
+            if not %%~nI == readline mklink %%~nxI %%I
```

```suggestion
           mklink %%~nxI %%I
```
Not installing readline.